packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -460,7 +460,7 @@ class _SnackBarState extends State<SnackBar> {
     final Color themeBackgroundColor = isThemeDark
       ? colorScheme.onSurface
       : Color.alphaBlend(colorScheme.onSurface.withOpacity(0.80), colorScheme.surface);
-    final ThemeData inverseTheme = ThemeData(
+    final ThemeData inverseTheme = theme.copyWith(
       brightness: brightness,
       backgroundColor: themeBackgroundColor,
```

Comment:
Can we have the Snackbar use the colorScheme's background color instead of ThemeData.backgroundColor?

Reply:
done